server/tests/mock-data/users.json

```diff
@@ -2,7 +2,17 @@
     {
         "id": 1,
         "username": "foobar",
-        "fullname": "Foo Bar",
-        "email": "foo@bar.com"
+        "roles": [
+            {
+                "name": "admin"
+            },
+            {
+                "name": "tester"
+            }
+        ]
```

Comment:
`admin` and `tester` seems to be a standard we're sticking with going forward but accounting for additional entries/renames shouldn't be a problem. It will be up to the server to have a layer to resolve accordingly.

Reply:
Are you saying that turning the objects into strings should happen outside Sequelize? If so that makes sense, but I just want to make sure I understand!